inkcpp_test/Globals.cpp

```diff
@@ -0,0 +1,68 @@
+#include "catch.hpp"
+
+#include <story.h>
+#include <globals.h>
+#include <runner.h>
+
+using namespace ink::runtime;
+
+SCENARIO("run story with global variable", "[global variables]")
+{
+	GIVEN ("a story with global variables")
+	{
+		story* ink = story::from_file("ink/GlobalStory.bin");
```

Comment:
I actually just merged in a PR that makes sure the Inklecate compiler is available on all OSs so you can go back to loading from the ink file directly. 

https://github.com/brwarner/inkcpp/pull/21

Reply:
nice